impl/pom.xml

```diff
@@ -48,16 +48,6 @@
         <dependency>
             <groupId>org.apache.zookeeper</groupId>
             <artifactId>zookeeper</artifactId>
-            <exclusions>
```

Comment:
Curious why you made this change?

Reply:
I moved the exclusion up to dependency management as looking at the dependency graph I was seeing both log4j 1.x and 2.x in the graph.